src/toolchain/export-android.md

```diff
@@ -0,0 +1,105 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+NOTE: Keep track of where you install the Android Command Line Tools and JDK 17
+
+Exporting with gdext for godot requires some of the same pieces that are required for building godot from source.
+Specifically, the Android SDK Command Line Tools and JDK 17 as mentioned in Godot's documentation here: 
+https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#requirements
+
+Once you have those installed, you then need to follow godot's instructions for setting up the build system here:
+https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#setting-up-the-buildsystem
+
+As of writing this, I'm not entirely sure if JDK 17 is required in order for Android Command Line Tools to function properly.
+I have not tested without it.  To be safe, it is included in the documentation for now.
+
+Now on to the fun part of actually compiling gdext for Android! :)
+
+Set the environment variable CLANG_PATH to point to Android's build of clang.  Example:
+
+CLANG_PATH is used by bindgen's clang-sys dependency.  Documentation here: https://github.com/KyleMayes/clang-sys?tab=readme-ov-file#environment-variables 
+
+```bash
+#!/bin/bash
+
+export CLANG_PATH = $"[android-cli-directory]/[android-cli-version]/ndk/[ndk-version]/toolchains/llvm/prebuilt/[host-machine-os]/bin/clang"
+```
```

Comment:
Use links in text when possible. Use curly braces and camelCase for placeholders.

Also write some text that puts the bash code block into context. The `#!/bin/bash` is not needed since it's not a full script, just a snippet.
```suggestion
`CLANG_PATH` is used by bindgen's clang-sys dependency. See also [clang-sys documentation][clang-sys-docs].
<some text introducing the bash section>:

```bash
export CLANG_PATH = $"{androidCliDirectory}/{androidCliVersion}/ndk/{ndkVersion}/toolchains/llvm/prebuilt/{hostMachineOs}/bin/clang"
```

Reply:
And at the end of the page or section:
```
[clang-sys-docs]: https://github.com/KyleMayes/clang-sys?tab=readme-ov-file#environment-variables 
```